Czaple/Attractions/Screens/AttractionList.swift

```diff
@@ -30,15 +37,24 @@ class AttractionList: UITableViewController {
             ( currentLocation = locManager.location ) : ( currentLocation = CLLocation(latitude: 0, longitude: 0) )
         navigationController?.isNavigationBarHidden = false
     }
+    // MARK: - Navigation
 
-    // MARK: - Table view data source
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+
+}
+
+extension AttractionList {
+// MARK: - Table view data source
 
     override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
you can remove this method because it does not change default implementation ( returns 1)